### DISCUSSION TITLE
关于在飞桨云端环境运行from ipywidgets import interact报错ModuleNotFoundError: No module named 'ipywidgets'

### DISCUSSION THREAD

**Discussion by AdvancedRoman** [datawhalechina/learn-python-the-smart-way-v2#29] [提问答疑]
请问我在飞桨云端环境运行
```
from ipywidgets import interact
import ipywidgets as widgets
def f(x, y):
    print(f"A simple math question: \"{x} + {y} = ?\", the answer is {x + y}!")
interact(f,x=10, y=20)
```
为什么报错？需要配置什么条件或环境吗？
![image](https://github.com/datawhalechina/learn-python-the-smart-way-v2/assets/49369347/f581a4ae-8099-41a4-8f67-14e64401154e)
为什么在本地jupyter运行上面代码会一次就成功？
![image](https://github.com/datawhalechina/learn-python-the-smart-way-v2/assets/49369347/5b0cb8b9-2842-40c3-89f4-117e89151a8e)

**Comment by limafang** [comment]
请仔细阅读报错信息



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2023年12月11日 23:37 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***> |
| 主题 | [datawhalechina/learn-python-the-smart-way-v2] 关于在飞桨云端环境运行from ipywidgets import interact报错ModuleNotFoundError: No module named 'ipywidgets' (Discussion #29) |

请问我在飞桨云端环境运行

from ipywidgets import interact
import ipywidgets as widgets
def f(x, y):
    print(f"A simple math question: \"{x} + {y} = ?\", the answer is {x + y}!")
interact(f,x=10, y=20)


为什么报错？需要配置什么条件或环境吗？
image.png (view on web)
为什么在本地jupyter运行上面代码会一次就成功？
image.png (view on web)
